boot/assets.go

```diff
@@ -421,12 +426,12 @@ type TrustedAssetsUpdateObserver struct {
 	model *asserts.Model
 
 	bootBootloader    bootloader.Bootloader
-	bootTrustedAssets []string
+	bootTrustedAssets map[string]string
 	bootManagedAssets []string
 	changedAssets     []*trackedAsset
 
 	seedBootloader    bootloader.Bootloader
-	seedTrustedAssets []string
+	seedTrustedAssets map[string]string
```

Comment:
same,  another approach is to define a type for this and explain the two side in the type definition